src/validation.ts

```diff
@@ -675,7 +676,9 @@ export function discriminatedUnion<
     return validator.validate(v, context, p)
   })
 
+  discriminated.meta.discriminatedUnion = union(...vs).union
```

Comment:
Why not just `discriminated.meta.discriminatedUnion = vs` ? You just want the list of validators used in the discriminated union, right?

Reply:
Indeed, you are right, updated accordingly.